docs/installing-solidity.rst

```diff
@@ -9,11 +9,18 @@ Installing the Solidity Compiler
 Versioning
 ==========
 
-Solidity versions follow `semantic versioning <https://semver.org>`_ and in addition to
-releases, **nightly development builds** are also made available.  The nightly builds
-are not guaranteed to be working and despite best efforts they might contain undocumented
-and/or broken changes. We recommend using the latest release. Package installers below
-will use the latest release.
+Solidity versions mostly follow `Semantic Versioning <https://semver.org>`_.  We make
+an assurance in addition to SemVer that we will not introduce breaking changes into a patch
+level release. That means if you are using 0.x.y, then you can expect everything to work
+the same or better with 0.x.z with z > y.
+
+The nightly builds are not guaranteed to be working and despite best efforts they might
```

Comment:
I think it would be nice to frame this a bit more positively - nightly builds are usually very stable.

Reply:
It's actually pretty much unchanged from how it was before. It was just moved :) But I agree that it could be framed better. How about this:

```rst
in addition to releases, we provide **nightly development builds** with the intention of making it easy
for developers to try out upcoming features and provide early feedback. Note, however, that while
the nightly builds are usually very stable, they contain bleeding-edge code from the development branch
and are not guaranteed to be always working. Despite our best efforts, they might contain undocumented
and/or broken changes that will not become a part of an actual release. They are not meant for
production use.
```

Reply:
Thank you @cameel  added at f71b3de3d93598f534f15f660235c4bd3c78b5bd.

Looks great